src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
-    public void getStatistic(String fromFileName, String toFileName) {
+    private static final String SIGN = "\\W+";
+    private static final String SPACE = " ";
+    private static final String SEPARATOR = System.lineSeparator();
+
+    public static void getStatistic(String fromFileName, String toFileName) {
+        WorkWithFile workWithFile = new WorkWithFile();
+        String[] splitLines = workWithFile.readLines(fromFileName);
+        String calculatedReport = workWithFile.calculateArray(splitLines);
+        workWithFile.writeToFile(toFileName, calculatedReport);
```

Comment:
We don't need to create instance of class inside methods of this class
```suggestion
        String[] splitLines = readLines(fromFileName);
        String calculatedReport = calculateArray(splitLines);
        writeToFile(toFileName, calculatedReport);
```